libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ErrorResponse.java

```diff
@@ -106,7 +104,10 @@ public enum ErrorResponse {
     UNEXPECTED_CREDENTIAL_TYPE(1092, "Unexpected credential type"),
     IPV_SESSION_NOT_FOUND(
             1093, "Invalid IPV session id provided, corresponding session not found in db"),
-    CLIENT_OAUTH_SESSION_NOT_FOUND(1093, "Client OAuth session not found");
+    CLIENT_OAUTH_SESSION_NOT_FOUND(1094, "Client OAuth session not found"),
+    MISSING_LANGUAGE(1095, "Missing language choice from the frontend"),
+    FAILED_TO_VALIDATE_VERIFIABLE_CREDENTIAL_RESPONSE(
+            1096, "Failed to validate verifiable credential response");
```

Comment:
@blakeyp I'm not sure what these error codes are used for in terms of reporting etc - it looks like 1051 has been duplicated for some time. Is there any problem we with deduplicating it now? (to a lesser extent 1093 too)

Reply:
Also, worth noting there are some gaps in the iterated codes

Reply:
Not so worried about gaps but yeah let's deduplicate (we aren't relying on these specific error codes in any reporting)